src/vision/ros_nmpt_saliency/src/salient_feature_tracker.py

```diff
@@ -46,11 +53,15 @@ def lookat_salientP_cb(self, data):
     else:
         loc.y =(0.5 -loc.y)
     t.z = loc.y
-
-    if self.GLOBAL_FLAG and degree >=7: # Look to Certainly Determined Salient Point
+    
+    if self.GLOBAL_FLAG and degree >=13: # Look to Certainly Determined Salient Point
         self.look_pub.publish(t)
         self.look_gaze.publish(t)
+        locate = "(StateLink (AnchorNode \"locations\")" + \
+                "(ListLink (NumberNode {})))\n".format(loc)
 
+        netcat(self.hostname, self.port, locate + "\n")
```

Comment:
Same comment here as above: the sound/audioclass.py file should subscribe to the ros events, and then place the appropriate atoms into the atomspace.